airflow/ui/src/components/ui/index.ts

```diff
@@ -0,0 +1,32 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export { Dialog } from "./Dialog";
+export { Pagination } from "./Pagination";
+export { Select } from "./Select";
+export { Alert } from "./Alert";
+export { CloseButton } from "./CloseButton";
+export { InputGroup, type InputGroupProps } from "./InputGroup";
+
+export { Switch } from "./Switch";
+export { Tag } from "./Tag";
+export { Tooltip } from "./Tooltip";
+
+export { ProgressBar } from "./ProgressBar";
+export { Menu } from "./Menu";
```

Comment:
This file might benefit from using wildcard exports, as it is going to continue to grow and input typing exports.
```suggestion
export * from "./Dialog";
export * from "./Pagination";
export * from "./Select";
export * from "./Alert";
export * from "./CloseButton";
export * from "./InputGroup";

export * from "./Switch";
export * from "./Tag";
export *from "./Tooltip";

export * from "./ProgressBar";
export * from "./Menu";
```